### PR TITLE
[helm] add remote logging option

### PR DESCRIPTION
**Pull request by sherry-x**
Add option to push logs to remote location

### CODE REVIEW

File: terraform/helm/aptos-node/templates/validator.yaml [hunk 1]
```diff
@@ -208,4 +212,8 @@ spec:
   - ports:
     - protocol: TCP
       port: 443
+  # Logging
+  - ports:
+    - protocol: TCP
+      port: 5044
```
Comment (rustielin) [inline terraform/helm/aptos-node/templates/validator.yaml:218]: this assumes struct log shipper is at 5044. maybe use helm values to be able to configure struct log shipper address and port?

File: terraform/helm/aptos-node/values.yaml [hunk 2]
```diff
@@ -23,6 +23,7 @@ validator:
     size: 350Gi
   rust_log: info
   rust_log_remote: debug,hyper=off
+  remoteLogAddress:
```
Comment (rustielin) [inline terraform/helm/aptos-node/values.yaml:26]: see comment above wrt port number assumption

File: terraform/helm/aptos-node/templates/validator.yaml (hunk 1 above)
Comment (sherry-x) [reply terraform/helm/aptos-node/templates/validator.yaml:218]: we didn't make the port configurable on the logger helm chart yet. would be a good followup

Comment (sherry-x) [reply terraform/helm/aptos-node/templates/validator.yaml:218]: maybe even make a good first issue for this

**Review by rustielin — APPROVED** [review]
LGTM assuming logger defaults (see comments)

**Pull request merged**